src/imitation/rewards/reward_nets.py

```diff
@@ -49,6 +50,25 @@ def forward(
     ):
         """Compute rewards for a batch of transitions and keep gradients."""
 
+    def preprocess(
```

Comment:
Mentioned this elsewhere, but am trying to wrap my head around why this should be a separate method, and not just rolled into the forward pass?